Aptoide/aptoide-client-v8
# [DT-340] Implement cache bypass for home bundle fetch methods

**What does this PR do?**

   Implements the logic to fetch home bundles and app lists with cache bypass.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] BundlesViewModel.kt

**How should this be manually tested?**

  - Check the logic to bypass the cache in the bundle fetching.
  - Use a HTTP proxy like Charles Proxy to check if the responses from pull to refresh are bypassing the cache.
  - An OkHttp interceptor can be used to check if the responses come from cache or network. (check docs)

**What are the relevant tickets?**

  Tickets related to this pull-request: [DT-340](https://aptoide.atlassian.net/browse/DT-340)

**Code Review Checklist**
